### DISCUSSION TITLE
Upgrade Flux to `v2.1.2`

### DISCUSSION THREAD

**Discussion by lrdplopes** [fluxcd/flux2#4636] [General] [answered]
Good day, Flux community.
I need to upgrade the Flux version of my organization to specific `v2.1.2`. However, when I try to `bootstrap` it, I get the message:
```
flux bootstrap git --version=v2.1.2 --url=ssh://git@github.com/<organization>/flux --branch=main --path=clusters/<my-cluster> --private-key-file=<my-private-key>

✗ targeted version 'v2.1.2' is not compatible with your current version of flux (2.2.3)
```

How do I deal with my Flux version? Thank you very much for any help you can provide.
Cheers.

**Comment by stefanprodan** [comment, marked as answer]
Download the CLI binary at the version you want to upgrade to from the release page: https://github.com/fluxcd/flux2/releases

**Reply by lrdplopes** [reply]
Thanks for your quick message, @stefanprodan.

Sure, I've done that. I even tried manually installing this version in my local to proceed with the upgrade. However, it seems my Flux version still with:
```
flux version
flux: v2.2.3
```

Can you please guide me with the correct steps? I so much appreciate it in advance.

**Reply by stefanprodan** [reply]
Run the binary you downloaded not the one in your path.

**Reply by lrdplopes** [reply]
Awesome. I will try that. Thank you so much.
